src/main/java/com/autotune/Autotune.java

```diff
@@ -112,7 +122,11 @@ public static void main(String[] args) {
                 // load available datasources from db
                 loadDataSourcesFromDB();
                 // setting up DataSources
-                setUpDataSources();
+                try {
+                    setUpDataSources();
+                } catch (Exception e) {
+                    LOGGER.error("***Failed to setup datasource connection due to : {}***", e.getMessage());
```

Comment:
Please have all of the constants defined and moved to appropriate place. I see several such messages all throughout this PR

Reply:
done